meteor/override-fastclick.js

```diff
@@ -0,0 +1,6 @@
+// disable fastclick package in Meteor if included
```

Comment:
ohh I see, then I can place it inside the package.js I think.
Or in any other place you want me to